stable/postgresql/templates/statefulset-slaves.yaml

```diff
@@ -77,17 +77,24 @@ spec:
           {{- end }}
           command:
             - /bin/sh
-            - -c
+            - -cx
             - |
+              echo "current user id: `id`"
               mkdir -p {{ .Values.persistence.mountPath }}/data
               chmod 700 {{ .Values.persistence.mountPath }}/data
-              find {{ .Values.persistence.mountPath }} -mindepth 0 -maxdepth 1 -not -name ".snapshot" -not -name "lost+found" | \
+              find {{ .Values.persistence.mountPath }} -mindepth 1 -maxdepth 1 -not -name ".snapshot" -not -name "lost+found" | \
+              {{- if and .Values.volumePermissions.securityContext.enabled ( not (eq ( toString ( .Values.securityContext.runAsUser ) ) "auto")) }}
                 xargs chown -R {{ .Values.securityContext.runAsUser }}:{{ .Values.securityContext.fsGroup }}
-              {{- if .Values.shmVolume.enabled }}
+              {{- else }}
+                xargs chown -R `id -u`:`id -G | cut -d " " -f2`
```

Comment:
is there any specific reason to use `id -G | cut -d " " -f2` instead of `id -g`?

Reply:
yes, because we need the 2nd group...
1st group is root, and it's the one we have to avoid in such hardened envs (we don't have perms to run with it).

Reply:
/solved